chat/models.tsp

```diff
@@ -293,16 +267,16 @@ model ChatCompletionNamedToolChoice {
 }
 
 /**
- * Controls which (if any) function is called by the model.
- * `none` means the model will not call a function and instead generates a message.
- * `auto` means the model can pick between generating a message or calling a function.
- * Specifying a particular function via `{"type": "function", "function": {"name": "my_function"}}`
- * forces the model to call that function.
- *
- * `none` is the default when no functions are present. `auto` is the default if functions are
- * present.
+ * Controls which (if any) tool is called by the model.
+ * `none` means the model will not call any tool and instead generates a message.
+ * `auto` means the model can pick between generating a message or calling one or more tools.
+ * `required` means the model must call one or more tools.
+ * Specifying a particular tool via `{"type": "function", "function": {"name": "my_function"}}`
+ * forces the model to call that tool.
+ * 
+ * `none` is the default when no tools are present. `auto` is the default if tools are present.
  */
-alias ChatCompletionToolChoiceOption = "none" | "auto" | ChatCompletionNamedToolChoice;
+alias ChatCompletionToolChoiceOption = "none" | "auto" | "required" | ChatCompletionNamedToolChoice;
```

Comment:
```suggestion
@extension("x-oaiExpandable", true)
union ChatCompletionToolChoiceOption {
  "none",
  "auto",
  "required",
  ChatCompletionNamedToolChoice
}
```

We'll need to turn this into an union if we want to add `x-oaiExpandable`
![image](https://github.com/joseharriaga/openai-in-typespec/assets/21109913/d9ac54a9-800c-4e3b-a842-b9003bc2dc4c)